### ISSUE TITLE
FGA not recognizing the AP refill screen

### ISSUE THREAD

**Issue by GourmetNewell** [Fate-Grand-Automata/FGA#1455]
### Preparation

- [X] I tested the [latest release](https://github.com/Fate-Grand-Automata/FGA/releases/tag/canary)
- [X] I looked at other issues (even the [closed ones](https://github.com/Fate-Grand-Automata/FGA/issues?q=is%3Aissue+is%3Aclosed))
- [X] I read the [Troubleshooting Guide](https://github.com/MathewSachin/Fate-Grand-Automata/wiki/Troubleshooting)

### FGO server

JP

### FGA build number

1892

### Describe the bug

As pictured, FGA doesnt recognize the screen when at the end of a round you need to recharge your AP. It does select a refill method (golden apple, silver, bronze, etc) but after that it falls flat on its face. 
![Screenshot_20221120-204321](https://user-images.githubusercontent.com/55465083/202952385-d5f695a9-5faa-4e97-b15d-bb323e288539.png)
Here is the screen on debug mode

### Video

No recording to show since the app still bugs out whenever I try to record my screen with FGA running. I turn on screen recording then activate FGA and then a second later FGA quits with the following error:

![Screenshot_20221118-193714](https://user-images.githubusercontent.com/55465083/202952500-7a132f6a-2957-480a-8a7f-78d39740dfa5.png)

### Device model

Google Pixel 6

### Android version

13

### Screen size

1280x720

### RAM

8GB

**Comment by GourmetNewell** [comment]
Just tried it out now. Curiously enough, if I do start FGA at the start quest screen and have the refill method set to something (Golden apple, silver, etc) it DOES select the right apple and refill then start the quest for me. But after it completes said quest, it gets stuck at the refill screen again. It's almost as if the app stops working at a certain point which would explain why it stops recognizing face cards full stop - curious that it is still able to activate skills and noble phantasms though.

**Comment by reconman** [comment]
I'm thinking this is related to #1452.

Using Skill speedup seems to mess up touch screen inputs for FGO JP, so some clicks are lost. I'll build a version with Skill Speedup as an optional feature so I don't get complaints about FGO JP every week.

In FGO NA, I don't have any problems with missed inputs.

**Comment by reconman** [comment]
Please try https://github.com/Fate-Grand-Automata/FGA/suites/9455560943/artifacts/447475628

**Comment by GourmetNewell** [comment]
Sorry, but looks like this version has the same issue. I tried to make another recording but we're back to the "Invalid Media Projection" error this time. Even with skill speed-up disabled (which it is by default, it seems), the cards stop getting recognized as well as the refill screen - back to the issue of FGA appearing to "stop working". 

Out of curiosity I tried it with NA but the issue is present there as well. Build 1856 still works.
